test/lib.similar.js

```diff
@@ -8,4 +8,8 @@ describe('Similar method', () => {
     return store.similar({id: '553834731'})
     .then((apps) => apps.map(assertValidApp));
   });
+  it('should fetch a valid application list in fr country', () => {
```

Comment:
No the similar app of a french-lang app can be a english-lang app.

I will add specific test, that it is diffferent from default store.